javaslang-benchmark/src/test/java/javaslang/BenchmarkPerformanceReporter.java

```diff
@@ -162,15 +171,15 @@ public void print() {
         private void printHeader() {
             final String alternativeImplHeader = alternativeImplementations.map(altImpl -> padRight(altImpl, altImplColSize(altImpl))).mkString("  ");
             final String header = String.format("%s  %s  %s  %s  %s  %s  ±%s %s  %s",
-                    padLeft("Target", groupSize),
-                    padLeft("Operation", nameSize),
-                    padLeft("Impl", implSize),
-                    padRight("Params", paramKeySize),
-                    padRight("Count", countSize),
-                    padRight("Score", scoreSize),
-                    padRight("Error", errorSize),
-                    padRight("Unit", unitSize),
-                    alternativeImplHeader
+                                                padLeft("Target", groupSize),
+                                                padLeft("Operation", nameSize),
+                                                padLeft("Impl", implSize),
+                                                padRight("Params", paramKeySize),
+                                                padRight("Count", countSize),
+                                                padRight("Score", scoreSize),
+                                                padRight("Error", errorSize),
+                                                padRight("Unit", unitSize),
+                                                alternativeImplHeader
```

Comment:
Oh - I've overseen that the new formatter also adds here more whitespace. Could you please revert the formatter option in an additional PR. We want generally not add additional horizontal whitespace (like we had also before for method chaining).

It is minor for this PR. I will pull it in as-is (formatted). Once we have the new formatter I will format the whole code-base in a separate PR.

Sorry for being off-topic again :)